src/main/java/pugbot/core/entities/Game.java

```diff
@@ -19,437 +19,447 @@
 import pugbot.core.entities.settings.QueueSettingsManager;
 
 public class Game {
-	private Queue parentQueue;
```

Comment:
apologies for the changes in this file here, my ide is converting all the tabs to spaces automatically. i'll annotate the new changes